config.yml

```diff
@@ -3,13 +3,13 @@ customer: ""
 aml_experiment_details:
   compute_name: oor-testing
   env_name: oor-environment
-  env_version: 25
+  env_version: 26
   src_dir: "."
 
 azure_iot:
   hostname: "iothub-oor-ont-weu-itr-01.azure-devices.net"
   device_id: "test-diana"
-  shared_access_key: "{SHARED_ACCESS_KEY_IOT}"
+  shared_access_key: "SHARED_ACCESS_KEY_IOT"
```

Comment:
How do we deal with this? Either we commit with {..} and break the normal usage, or we commit without and break on edge. Finally time to split the repo? ;)

Reply:
Oops, definitely we need to split it 😆